cmd/ci-operator-prowgen/main.go

```diff
@@ -196,7 +299,14 @@ func generateJobs(
 
 	for _, element := range configSpec.Tests {
 		test := testDescription{Name: element.As, Target: element.As}
-		presubmits[orgrepo] = append(presubmits[orgrepo], *generatePresubmitForTest(test, repoInfo))
+		var podSpec *kubeapi.PodSpec
+		if len(element.From) != 0 || element.ContainerTestConfiguration != nil {
+			// TODO remove when the migration is completed
+			podSpec = generatePodSpec(repoInfo.org, repoInfo.repo, repoInfo.configFilename, test.Target)
+		} else {
+			podSpec = generatePodSpecTemplate(repoInfo.org, repoInfo.repo, repoInfo.configFilename, configSpec.ReleaseTagConfiguration.Name, &element)
```

Comment:
What happens when we don't have a `ReleaseTagConfiguration`?

Reply:
Good point.

We should add that as a validation to ci-operator. That's also the appropriate place for this (I'm surprised you let me get away with that ugly-as-all-hell regexp validation ; ):

https://github.com/openshift/ci-operator-prowgen/pull/4/files#diff-a30756393faedcdaae449d088f908ca4R190